components/Table/Table.js

```diff
@@ -42,40 +60,43 @@ export class Table {
       const cell = document.createElement('div');
       cell.classList.add('table-header__cell');
       cell.setAttribute('data-key', col.key);
-      cell.setAttribute('data-sortable', col.sortable);
+
       const headerSortBtn = document.createElement('div');
       headerSortBtn.classList.add('header-sort-btn');
 
       const actionsContainer = document.createElement('div');
       actionsContainer.classList.add('actions-container');
 
-      const arrowBtn = document.createElement('span');
-      arrowBtn.classList.add('material-icons');
-      arrowBtn.classList.add('arrow');
-      arrowBtn.textContent = 'south';
+      if (col.sortable) {
+        const arrowBtn = document.createElement('span');
+        arrowBtn.classList.add('material-icons');
+        arrowBtn.classList.add('arrow');
+        arrowBtn.textContent = 'south';
+        actionsContainer.append(arrowBtn);
 
-      const arrayOfListener = [headerSortBtn, arrowBtn];
+        const arrayOfActions = [headerSortBtn, arrowBtn];
 
-      if (col.sortable) {
-        arrayOfListener.forEach((el) =>
+        arrayOfActions.forEach((el) =>
           el.addEventListener('click', () => {
             const dataKey = cell.getAttribute('data-key');
+
             if (this.isOrderAsc === null) {
-              this.isOrderAsc = true;
-              this.sortCallback(dataKey, this.isOrderAsc);
-              arrowBtn.setAttribute('asc', this.isOrderAsc);
-              return;
-            }
-            if (this.isOrderAsc === true) {
-              this.isOrderAsc = false;
-              this.sortCallback(dataKey, this.isOrderAsc);
-              arrowBtn.setAttribute('asc', this.isOrderAsc);
-              return;
-            }
-            if (this.isOrderAsc === false) {
-              this.isOrderAsc = null;
-              this.initialArrCallback();
-              return;
+              if (this.callbacksMap.sortCallback) {
+                this.isOrderAsc = true;
+                this.callbacksMap.sortCallback(dataKey, this.isOrderAsc);
+              }
+              arrowBtn.setAttribute('data-asc', this.isOrderAsc);
+            } else if (this.isOrderAsc === true) {
+              if (this.callbacksMap.sortCallback) {
+                this.isOrderAsc = false;
+                this.callbacksMap.sortCallback(dataKey, this.isOrderAsc);
+              }
+              arrowBtn.setAttribute('data-asc', this.isOrderAsc);
+            } else {
+              if (this.callbacksMap.sortCallback) {
+                this.isOrderAsc = null;
+                this.callbacksMap.sortCallback();
+              }
```

Comment:
Not make sense to check `this.callbacksMap.sortCallback` three times
Just check it once